README.md

```diff
@@ -13,6 +13,10 @@ await _messageDialogService.ShowMessage(ct, mb => mb
 );
 ```
 
+## Preview
+From top to bottom: WinUI, Android, and iOS.
+![](doc/images/dialog.png)
```

Comment:
Can we have this image show the platforms horizontally rather than vertically?
Like here: https://github.com/nventive/UnoApplicationTemplate#preview